openmetadata-spec/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -860,6 +860,13 @@
         "sql"
       ],
       "additionalProperties": false
+    },
+    "tableProperties": {
+      "description": "Field to maintain the properties of table, for example. compression, file format etc.",
```

Comment:
instead of key value pairs, can we define what we want to capture?

Reply:
sure, have made the changes